lib/express/oidc.js

```diff
@@ -92,6 +105,13 @@ function config(app, { showLoginPage, idpConfig, serviceProvider }) {
         token_type: 'bearer',
         id_token: idToken,
       })
+
+      const now = Date.now()
+      nonceStore.forEach((value, key) => {
+        if (now > value.expires) {
+          nonceStore.delete(key)
+        }
+      })
```

Comment:
Is there a need to have this? Given that MockPass is meant for development purposes, I would assume that users would be happier to have long-living nonces so that their debugging is not disrupted by a potentially expiring nonce.

At any rate, it would probably be better to implement this through expiry-map or similar.

Reply:
In the event this is run as a long lived server instance not clearing the nonces leaks memory quite quickly. When the login page is repeatedly displayed it stores the full list of identities each time as the authorization codes are unique. 

I feel 5 minutes is already long enough, this just needs to be long enough for the user to select the user to login and for the relying party to call the token endpoint with the authorization code.

I'm not really sure it's worth pulling a dependency just for this though.

Reply:
Point taken, though I still think that given that MockPass is not actively maintained, in order to maintain readability, I would prefer to rely on a dependency to handle entry expiration, since arguably, that's ancillary to the main focus of the codebase - pretending to be SingPass/CorpPass/MyInfo.

Reply:
Ok. Have changed to use expiry-map instead.